CONTRIBUTING.md

```diff
@@ -0,0 +1,227 @@
+# Contributing
+
+## Licensing
+
+Code contributions ard licensed under the [3-Clause BSD](https://opensource.org/licenses/BSD-3-Clause) license.
+
+Content contributions are licensed under the
+[Creative Commons CC BY 3.0](https://creativecommons.org/licenses/by/3.0/) license.
+
+## Process/Workflow
+
+The general contribution workflow is for contributors to fork the project repository, work in a branch created off of
+main, and submit a pull request against the project repo's main branch to merge their contributions.
```

Comment:
Should "main" be formatted differently since it's also the name of the branch. Maybe `main` or "Main".

Reply:
I'll put main in backticks. I think the names are case sensitive so I'd prefer not to do the capitalization.